README.md

```diff
@@ -1,2 +1,28 @@
 # stoa
 API server for Agora
+
+
+
+## Build instructions
+
+# Install the node.Js >= 12.18.0 (includes npm 6.14.4) (you might want to use a newer version)
```

Comment:
@TrustHenry the readme says it requires node.js >= 12.18.0.

So then it's unnecessary to test node.js v10, right? We might want to disable it from the CI.

Reply:
We actually need to test the previous version.
We recommend more than the latest version here, but I would like to describe it as a guideline.
When I tried, Node 8 found a problem with the mocha test.
I want to support at least 10 versions.

Reply:
Oh ok, that's good!